<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QDialog" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iRobot Order Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QLabel" name="iRobotLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>iRobot Order Form</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="robotAGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="robotANameLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="robotASpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>25</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="robotAComboBox">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>25</y>
      <width>210</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="robotAPicLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="robotAPriceLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="robotASubtotalBox">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>25</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="HLineTwo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>515</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="addressGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>600</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="yesCheckBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>33</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QLabel" name="sameAsLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>35</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Same as biling address:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="noCheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>33</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shippingAddressLineOne">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>270</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="shippingAddressLineTwo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>270</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="shippingAddressLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Shipping Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="billingAddressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Billing Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="billingAddressLineOne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>270</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="billingAddressLineTwo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>270</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="addressVLine">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>3</width>
      <height>100</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="creditCardGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="creditCardTypeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Credit Card : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="creditCardTypeBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="creditCardNumberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Card Number : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="creditCardNumberLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>225</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="creditCardNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name on Card : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="creditCardNameLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>225</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="creditCardExpireLabel">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>40</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date Expires : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateMonthLine">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateSlash">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>40</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateYearLine">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>40</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="creditCardCvvLabel">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>70</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CVV : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cvvLine">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="HLineOne">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>385</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>640</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="placeOrderButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>640</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Place Order</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subtotalLine">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>290</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="shippingLine">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>315</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>100.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="subtotalLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Subtotal : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="taxLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tax (7.75%) : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ShippingLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>315</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Shipping &amp; Handling : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="totalPriceLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>365</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TOTAL PRICE : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taxLine">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>340</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalPriceLine">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>365</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="robotBGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="robotBNameLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="robotBSpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>25</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="robotBComboBox">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>25</y>
      <width>210</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="robotBPicLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="robotBPriceLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="robotBSubtotalBox">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>25</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="robotCGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="robotCNameLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="robotCSpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>25</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="robotCComboBox">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>25</y>
      <width>210</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="robotCPicLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="robotCPriceLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="robotCSubtotalBox">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>25</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
